config/symbols.us.stno1.txt

```diff
@@ -46,6 +48,7 @@ D_80182488 = 0x801827D8;
 D_801824B8 = 0x80182808;
 D_801824C0 = 0x80182810;
 D_80182A4C = 0x80182A4C;
+sprites_3 = 0x801A86BC;
```

Comment:
This is called `sprites_nz0_3` in axe knight, but I think this will probably need to be changed to use OVL_EXPORT or something similar in future once the other overlay sprite banks get imported. I've just named this a bit more generically for now and mapped the symbol direct